engine/graphics/src/vulkan/graphics_vulkan.cpp

```diff
@@ -774,6 +774,11 @@ namespace dmGraphics
         return true;
     }
 
+    static PipelineState VulkanGetPipelineState(HContext context)
+    {
+        return context->m_PipelineState;
+    }
+
```

Comment:
Bugfix, was apparently missing.. Would be nice to have unit tests for all graphics pipelines "somehow"